client/src/components/BuildingDetails.js

```diff
@@ -1,39 +1,37 @@
 import React from 'react';
-import { Drawer, Typography, Row, Col, Icon } from 'antd';
+import { Typography, Row, Col, Icon } from 'antd';
 import { Trans } from '@lingui/macro';
 
 const { Title } = Typography;
 
 export default function BuildingDetails(props) {
-  const { visible, onClose, details } = props;
+  const { onClose, details } = props;
   return (
-    <Drawer
-      placement="right"
-      mask={false}
-      visible={visible}
-      onClose={onClose}
-      getContainer={false}
-      style={{ position: 'absolute' }}
-    >
-      <Title className="building-details-title" level={3}>
-        <Row>
+    <Row className="building-details">
+      <Row>
+        <Title className="building-details-title" level={3}>
           <Col lg={1} span={0}>
             <Icon type="environment" />
           </Col>
-          <Col lg={{ span: 22, offset: 1 }} span={24}>
+          <Col lg={{ span: 21, offset: 1 }} span={23}>
             <Trans>Building Info</Trans>
           </Col>
-        </Row>
-      </Title>
+          <Col span={1}>
+            <Icon type="close" onClick={onClose} />
+          </Col>
+        </Title>
+      </Row>
       {details ? (
         <div>
           <Row>
             <Col lg={{ span: 13, offset: 2 }} span={13}>
               <Trans>Address</Trans>:
             </Col>
-            <Col lg={9} span={11}>
-              {details.address} <Trans>no.</Trans> {details.post_code}
-            </Col>
+            {details.address && details.post_code && (
```

Comment:
do we want to still show the `address` but only hide "no." text if the `post_code` is `null`?
```suggestion
{details.address} {details.post_code && <Trans>no.</Trans>} {details.post_code}
```

Reply:
Initially I have added the null check, because while the details were loading the "no." text appeared in the middle of nowhere (see attached image). I don't know if `post_code` is a required field or not though, so I have added the separate check, as you suggested. :)

<img width="464" alt="Screenshot 2021-03-20 at 21 19 12" src="https://user-images.githubusercontent.com/17540230/111883120-fcecc180-89c1-11eb-9882-14e00ca347e0.png">